python/src/CMakeLists.txt

```diff
@@ -1002,6 +1005,19 @@ if (SPHINX_FOUND)
   foreach (file ${EXTRA_PYTHON_FILES})
     execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/${file} ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/${file})
   endforeach ()
+  
+  message("****")
+  message( ${CMAKE_CURRENT_SOURCE_DIR})  
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/chaboche_model.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/chaboche_model.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/ackley_function.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/ackley_function.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/branin_function.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/branin_function.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/cantilever_beam.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/cantilever_beam.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/deflection_tube.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/deflection_tube.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/flood_model.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/flood_model.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/ishigami_function.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/ishigami_function.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/logistic_model.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/logistic_model.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/stressed_beam.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/stressed_beam.py)
+  execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/usecases/viscous_free_fall.py ${OPENTURNS_PYTHON_BUILD_MODULE_PATH}/openturns/usecases/viscous_free_fall.py)
```

Comment:
we may use the "files (glob)" command to list these, then copy them in a loop

Reply:
@jschueller Done ! Thanks for pointing  it out.